xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -1233,9 +1270,58 @@ CPVREpgInfoTagPtr CPVRTimerInfoTag::GetEpgInfoTag(bool bCreate /* = true */) con
       }
     }
   }
+
+  if (!m_epgTag && HasFallbackEpgData() && !IsTimerRule())
+  {
+    CPVRChannelPtr channel(m_channel);
+    if (!channel)
+    {
+      channel = CServiceBroker::GetPVRManager().ChannelGroups()->Get(m_bIsRadio)->GetGroupAll()->GetByUniqueID(m_iClientChannelUid, m_iClientId);
+
+      CSingleLock lock(m_critSection);
+      m_channel = channel;
+    }
+
+    if (channel)
+    {    
+      m_epgTag = CreateFallbackEpgInfoTag();
+    }
+  }  
   return m_epgTag;
 }
 
+bool CPVRTimerInfoTag::HasFallbackEpgData() const
+{
+  return !m_strPlotOutline.empty() || !m_strPlot.empty();
+}
+
+CPVREpgInfoTagPtr CPVRTimerInfoTag::CreateFallbackEpgInfoTag() const
+{
+  EPG_TAG tag = {0};
+  tag.strPlotOutline = m_strPlot.c_str();
+  tag.strPlot = m_strPlotOutline.c_str();
+  tag.strTitle = m_strTitle.c_str();
+  tag.iUniqueChannelId = m_iClientChannelUid;
+  tag.iUniqueBroadcastId = m_iEpgUid;
+  tag.iYear = m_iYear;
+  tag.iGenreType = m_iGenreType;
+  tag.iGenreSubType = m_iGenreSubType;
+  tag.strGenreDescription = m_strGenreDescription.c_str();
+  tag.iSeriesNumber = m_iSeriesNumber;
+  tag.iEpisodeNumber = m_iEpisodeNumber;
+  tag.iEpisodePartNumber = m_iEpisodePartNumber;
+
+  time_t startTime;
+  m_StartTime.GetAsTime(startTime);
+  tag.startTime = startTime;
```

Comment:
It is not guaranteed that timer start and stop time match epg event start and stop time!